docs/source/include.rst

```diff
@@ -0,0 +1,360 @@
+.. include:: ../onnx_doc_folder/onnx_aionnxml_ArrayFeatureExtractor.rst
```

Comment:
You should not need that file. The index is included all of them as soon as they referenced.